### ISSUE TITLE
Windows Store App Support

### ISSUE THREAD

**Issue by NVentimiglia** [jacobdufault/fullserializer#25]
I am getting errors when building for WindowsStoreApp. From the looks of it, the dll is inapropreatly trying to access System.Threading.Thread. Here are 2 of the exceptions, there are 231 in total.

````
Reference Rewriter: Error: type `System.Threading.Thread` doesn't exist in target framework. It is referenced from FullSerializer - Unity - WinRT.dll at System.Collections.Generic.IEnumerator`1<System.Object> FullSerializer.Internal.fsIDictionaryAdapter/<Iterate>d__1::System.Collections.Generic.IEnumerable<System.Object>.GetEnumerator().

UnityEngine.Debug:LogError(Object)
PostProcessMetroPlayer:RunReferenceRewriter(WSASDK, String, String, String, String) (at C:/buildslave/unity/build/PlatformDependent/MetroPlayer/Extensions/Managed/PostProcessMetroPlayer.cs:334)
PostProcessMetroPlayer:PostProcessSDKSpecific(WSASDK, BuildTarget, BuildOptions, String, String, ProjectImages, LibraryCollection) (at C:/buildslave/unity/build/PlatformDependent/MetroPlayer/Extensions/Managed/PostProcessMetroPlayer.cs:883)
PostProcessMetroPlayer:PostProcess(BuildTarget, BuildOptions, String, String, String, String) (at C:/buildslave/unity/build/PlatformDependent/MetroPlayer/Extensions/Managed/PostProcessMetroPlayer.cs:597)
UnityEditor.HostView:OnGUI()

````

````
Reference Rewriter: Error: type `System.AppDomain` doesn't exist in target framework. It is referenced from FullSerializer - Unity - WinRT.dll at System.Type FullSerializer.Internal.fsTypeLookup::GetType(System.String).

UnityEngine.Debug:LogError(Object)
PostProcessMetroPlayer:RunReferenceRewriter(WSASDK, String, String, String, String) (at C:/buildslave/unity/build/PlatformDependent/MetroPlayer/Extensions/Managed/PostProcessMetroPlayer.cs:334)
PostProcessMetroPlayer:PostProcessSDKSpecific(WSASDK, BuildTarget, BuildOptions, String, String, ProjectImages, LibraryCollection) (at C:/buildslave/unity/build/PlatformDependent/MetroPlayer/Extensions/Managed/PostProcessMetroPlayer.cs:883)
PostProcessMetroPlayer:PostProcess(BuildTarget, BuildOptions, String, String, String, String) (at C:/buildslave/unity/build/PlatformDependent/MetroPlayer/Extensions/Managed/PostProcessMetroPlayer.cs:597)
UnityEditor.HostView:OnGUI()
````

**Comment by NVentimiglia** [comment]
Going to close this issue.

I dont see any use of System.Threading. I don`t know why unity is identifying FullSerializer as the suspect.

**Comment by NVentimiglia** [comment]
Reopening this issue, it is FullSerializer. Steps to replicate.

- New Unity Project
- Add WinRT dll to Plugins
- Build and run as a WindowsStore App

**Comment by NVentimiglia** [comment]
I was able to get it to build by converting the WinRT library to a portable class library. The code mostly worked but did require some minor modifications.

**Comment by jacobdufault** [comment]
Fixed in f56c90af9d6e97c995248ccb8ed52e8df70cf125
